### PR TITLE
add issue and pull request templates

### LINKED ISSUE #709: Create issue and pull request templates
As part of the definition of the process to create and improve the Joyent design system, we should create a template for submitting issues and pull requests on this repo.

### PR DESCRIPTION
**Pull request by SaraVieira**
closes #709

### CODE REVIEW

**Comment by ramitos** [comment]
I would add a note about the expected commit format:

```
<type><(scope)?>: <msg>
```
where type is one of `build`, `chore`, `ci`, `docs`, `feat`, `fix`, `perf`, `refactor`, `revert`, `style`, or `test`.

and where scope is one of `ui-toolkit`, `my-joy-beta`, `cloudapi-gql`, `boilerplate`, and `create-instance`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+* **Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines
+- [ ] Tests for the changes have been added (for bug fixes / features)
+
+
+* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
+
+
+
+* **Does this PR Close a Issue?** (If not please create one)
```
Comment (yaili) [inline .github/PULL_REQUEST_TEMPLATE.md]: Does this PR close an issue?

**Comment by SaraVieira** [comment]
@yaili @ramitos You both right and updated ! 🚀

File: .github/COMMIT_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Commit Guidelines
+
+```
+<type><(scope)?>: <msg>
+```
+
+Where type is one of build, chore, ci, docs, feat, fix, perf, refactor, revert, style, or test.
```
Comment (ramitos) [inline .github/COMMIT_GUIDELINES.md]: s/where/what

Comment (SaraVieira) [reply .github/COMMIT_GUIDELINES.md]: What?

Comment (ramitos) [reply .github/COMMIT_GUIDELINES.md]: Oh, sorry, I understand that phrasing now. maybe adding `'s to the tokens would help

Comment (SaraVieira) [reply .github/COMMIT_GUIDELINES.md]: Where the type is one of
``` build, chore, ci, docs, feat, fix, perf, refactor, revert, style, or test.

Comment (SaraVieira) [reply .github/COMMIT_GUIDELINES.md]: better ?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (SaraVieira) [reply .github/PULL_REQUEST_TEMPLATE.md]: Fixed :)

**Pull request merged**